React/Views/RCTNavItemManager.m

```diff
@@ -20,10 +20,26 @@ - (UIView *)view
 }
 
 RCT_EXPORT_VIEW_PROPERTY(title, NSString)
+RCT_EXPORT_VIEW_PROPERTY(leftButtonTitle, NSString);
 RCT_EXPORT_VIEW_PROPERTY(rightButtonTitle, NSString);
 RCT_EXPORT_VIEW_PROPERTY(backButtonTitle, NSString);
 RCT_EXPORT_VIEW_PROPERTY(tintColor, UIColor);
 RCT_EXPORT_VIEW_PROPERTY(barTintColor, UIColor);
 RCT_EXPORT_VIEW_PROPERTY(titleTextColor, UIColor);
 
+- (void)set_leftButtonImageName:(id)json forView:(RCTNavItem *)view withDefaultView:(RCTNavItem *)defaultView
```

Comment:
These should use `RCT_CUSTOM_VIEW_PROPERTY` in case we change the prefix in the future, which is likely, e.g. to add an `rct_` prefix like you suggested ;).